omnibus/files/private-chef-cookbooks/private-chef/spec/libraries/helper_spec.rb

```diff
@@ -113,7 +113,40 @@
       end
     end
 
+    shared_context "elastic search version request fails" do
```

Comment:
Does `shared_context` let you share specs? I thought it was only for sharing test setup.